INSTALL.md

```diff
@@ -380,7 +380,7 @@ Compiler Warnings
     --strict-warnings
 
 This is a developer flag that switches on various compiler options recommended
-for OpenSSL development.  It only works when using gcc or clang as the compiler.
+for OpenSSL development.  It only works when using "gcc" or "clang" as the compiler.
```

Comment:
It was like this already at some places, and I just made it consistent here.
In these occurrences we do not talk about code portions but about compilers with those names.
Maybe it would be even better to drop the `"` around them in all occurrences?

Reply:
I don't feel strongly about this, so I will accept whatever you think it's best.

My rule of thumb usually is to use backticks for inline code, paths, and _commands_. That is why I would go for backticks here and elsewhere. But this is just personal preference.

Reply:
> I don't feel strongly about this, so I will accept whatever you think it's best.

So then I've just generally simplified "gcc" to gcc and "clang" to clang,
using the backticks e.g. `gcc` only where it is used to denote a config item name, which I'd consider as a king of code rather than the compiler name.